lib/sorbet-rails/routes_rbi_formatter.rb

```diff
@@ -4,62 +4,72 @@ class SorbetRails::RoutesRbiFormatter
 
   sig { void }
   def initialize
+    @parlour = T.let(Parlour::RbiGenerator.new, Parlour::RbiGenerator)
     @buffer = T.let([], T::Array[T.any(String, T::Array[String])])
```

Comment:
You probably don't need @buffer any more. It's all to prepare data for the `result` method